models/chainlink/arbitrum/chainlink_arbitrum_ccip_tokens_transferred_logs.sql

```diff
@@ -0,0 +1,49 @@
+{{
+  config(
+    
+    alias='ccip_tokens_transferred_logs',
+    materialized='view'
```

Comment:
We can add schema in the config blocks and change the materialized strategy to `incremental`.